pkg/jobrunaggregator/jobrunaggregatorapi/interfaces.go

```diff
@@ -0,0 +1,56 @@
+package jobrunaggregatorapi
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/openshift/ci-tools/pkg/junit"
+
+	goyaml "gopkg.in/yaml.v2"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+)
+
+// JobRunInfo is a way to interact with JobRuns and gather their junit results.
+// The backing store can vary by impl, but GCS buckets and
+type JobRunInfo interface {
+	IsFinished(ctx context.Context) bool
+
+	GetJobName() string
+	GetJobRunID() string
```

Comment:
There's a mix of JobRunID and JobRunName throughout the code, in this case it looks like ID is more accurate and easier for comprehension in relation to JobName. We can make this change later and I'd be happy to do it, but I'd like to standardize on ID.

Reply:
> There's a mix of JobRunID and JobRunName throughout the code, in this case it looks like ID is more accurate and easier for comprehension in relation to JobName. We can make this change later and I'd be happy to do it, but I'd like to standardize on ID.

No objection to consistency.  I don't feel strongly either way, but you cannot rename bigquery fields, so think carefully